frontend/.gitignore

```diff
@@ -11,13 +11,13 @@ node_modules
 dist
 dist-ssr
 *.local
+vite.config.ts
```

Comment:
Zeile 14:
Warum lässt Du die vite.config.ts ignorieren?
Sie wird zwar bei Dir trotzdem mitversioniert.
Aber in der .gitignore sollte sie sicherheitshalber nicht stehen.